crates/iota-e2e-tests/tests/dynamic_committee_tests.rs

```diff
@@ -122,16 +122,14 @@ impl StressTestRunner {
     }
 
     pub fn pick_random_active_validator(&mut self) -> IotaValidatorSummary {
-        let active_validators = match self.system_state() {
-            IotaSystemStateSummary::V1(v1) => v1.active_validators,
-            IotaSystemStateSummary::V2(v2) => v2.active_validators,
-            _ => panic!("unsupported IotaSystemStateSummary"),
-        };
-
-        active_validators
-            .get(self.rng.gen_range(0..active_validators.len()))
+        let system_state = self.system_state();
+        let n = system_state.iter_committee_members().count();
+        let random_validator = system_state
```

Comment:
```suggestion
        let random_comittee_member = system_state
```

Reply:
Thanks, addressed in [bc940d0](https://github.com/iotaledger/iota/pull/6015/commits/bc940d007168d3abebefc86ee33de3233c9bcb1b).